src/shared/actions/status.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Any actions related to the API/network status handling and messaging.
+ */
+import _ from 'lodash';
+import { createActions } from 'redux-actions';
+
+/**
+ * Export Actions for usage by Redux
+ */
+export default createActions({
+  STATUS: {
+    CLEAR_ALL_ERRORS_STATUS: _.noop(),
```

Comment:
Let's rename these to be `..._ERROR_ICON...`? Like `CLEAR_ALL_ERROR_ICONS`, `SET_NETWORK_ERROR_ICON`?
The other thing, I would like more to have a single set of `SET_ERROR_ICON`/`CLEAR_ERROR_ICON` methods, which in addition to `url` and `message` would take `id` of the icon. We define a constant set of valid IDs in the actions module. This way, when we want to add more icons for different errors we won't have to add actions / reducers code, just to add a new ID into the set and properly load the icon asset itself.

And, I mean, we still keep `CLEAR_ALL_ERROR_ICONS`.